Runtime/EcsactDefaults.cs

```diff
@@ -0,0 +1,59 @@
+using System.Runtime.CompilerServices;
+using System;
+
+#nullable enable
+
+[assembly: InternalsVisibleTo("EcsactRuntimeDefaults")]
+
+namespace Ecsact {
+	static public class Defaults {
+
+		internal static EcsactRuntime? _Runtime;
+		public static EcsactRuntime Runtime => _Runtime
+			?? throw new Exception(
+				"Runtime is null, if you want to access it as early as possible" +
+				" use Ecsact.Defaults.WhenReady"
+			);
```

Comment:
This would still fit in the 80 char limit right?
```suggestion
		public static EcsactRuntime Runtime => _Runtime ?? throw new Exception(
			"Runtime is null, if you want to access it as early as possible" +
			" use Ecsact.Defaults.WhenReady"
		);
```